### PR TITLE
DOCS-3206: Add FAQ article for disabling refunds

### PR DESCRIPTION
**Pull request by Joris-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

**Review by Daniel-MultiSafepay — APPROVED** [review]
The procedure is correct. I will @FrancescaNCook; or someone else from the docs team, check if the text properly complies with the style rules we use.

File: content/integration/ready-made/prestashop-1-7/faq/disabling-refunds.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title : "Disabling refunds"
+meta_title: "PrestaShop 1.7 - Disabling refunds - MultiSafepay Docs"
+read_more: "."
+url: '/prestashop-1-7/disable-refunds/'
+---
+**Disable refunds**
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/disabling-refunds.md:7]:
```suggestion
```

File: content/integration/ready-made/prestashop-1-7/faq/disabling-refunds.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+if you don't want refunds made from your PrestaShop [backend](/glossaries/multisafepay-glossary/#backend) to automatically process with the MultiSafepay API, you can disable this by following these steps:
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/disabling-refunds.md:9]:
```suggestion
By default, refunds initiated in your PrestaShop [backend](/glossaries/multisafepay-glossary/#backend) are automatically processed via our API.

 To disable this, follow these steps:
```

File: content/integration/ready-made/prestashop-1-7/faq/disabling-refunds.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+1. Sign in to your PrestaShop 1.7 backend.
+2. Go to **MultiSafepay module** > **Manage hooks**.
+3. Select **Display non-positionable hooks**.
+4. Find **actionOrderSlipAdd**, select the three dots and click on **Unhook**.
```
Comment (FrancescaNCook) [inline content/integration/ready-made/prestashop-1-7/faq/disabling-refunds.md:14]:
```suggestion
4. For **actionOrderSlipAdd**, select the three dots, and then click **Unhook**.
```

**Review by FrancescaNCook — APPROVED** [review]

**Pull request merged**